### PR TITLE
Update buttons in the status bar and check for updates

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

File: src/main/java/io/github/mzmine/gui/MZmineGUI.java [hunk 1]
```diff
@@ -614,13 +642,14 @@ public void displayMessage(String title, String msg) {
...
 
-      final Text text = new Text();
-//      text.getStyleClass().add("label");
-//      text.getStyleClass().add("text-id");
+      TextFlow flow = new TextFlow(new Text(msg));
+      if (url != null) {
+        Hyperlink href = new Hyperlink(url);
+        flow.getChildren().add(href);
+        href.setOnAction(event -> WebUtils.openURL(url));
+      }
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/gui/MZmineGUI.java:650]: does this still have the correct color in dark mode?

File: src/main/java/io/github/mzmine/gui/MZmineGUI.java [hunk 2]
```diff
@@ -592,18 +599,39 @@ public void setStatusBarText(String message, Color textColor) {
...
       }
-      statusBar.setText(message);
-      statusBar.setStyle("-fx-text-fill: " + FxColorUtil.colorToHex(textColor));
+      statusBar.setText(null);
+      if (statusLabel == null) {
+        statusLabel = new Label(message);
+        statusBar.getLeftItems().add(statusLabel);
+      } else {
+        statusLabel.setText(message);
+      }
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/gui/MZmineGUI.java:608]: the hack to remove the link could be creating a new label every time, but i don't think that's feasible. Why not have a method `public void setStatusBarText(String message, Color textColor, URL url)` that will put a url, and the normal `public void setStatusBarText(String message, Color textColor)` removes said url by default

**Review by SteffenHeu — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/mzmine/gui/MZmineGUI.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/gui/MZmineGUI.java:642]: I removed this hardcoded color. We now just use the default color. and the negative color

**Comment by robinschmid** [comment]
I have done exactly what you said. Good idea with one method. 
The colors are now the default style and the negative color if update needed. 
One method and the link is removed

**Review by SteffenHeu — APPROVED** [review]

**Comment by SteffenHeu** [comment]
thanks

**Pull request merged**